src/Tools/dotnet-dsrouter/DiagnosticsServerRouterCommands.cs

```diff
@@ -337,7 +337,7 @@ static string GetDefaultIpcServerPath(ILogger logger)
                 var path = Path.Combine(PidIpcEndpoint.IpcRootPath, $"dotnet-diagnostic-{processId}");
                 if (File.Exists(path))
                 {
-                    logger?.LogWarning($"Default IPC server path, {path}, already in use. To disable default diagnostics for dotnet-dsrouter, set COMPlus_EnableDiagnostics=0 and re-run.");
+                    logger?.LogWarning($"Default IPC server path, {path}, already in use. To disable default diagnostics for dotnet-dsrouter, set DOTNET_EnableDiagnostics=0 and re-run.");
```

Comment:
These directions will not work on .NET 6.0

Reply:
They are read from configuration system, which uses `DOTNET_` first and then falls back to `COMPlus_`, in .NET 6 as well: https://github.com/dotnet/runtime/blob/release/6.0/src/coreclr/inc/clrconfigvalues.h#LL164C35-L164C76

Reply:
Yes, I didn't look at the env var close enough; I thought it was one of the dump generation ones.